docs/UserGuide.md

```diff
@@ -282,12 +312,17 @@ Format: `add_task td/TASK_DESCRIPTION d/DEADLINE ev/EVENT_NAME`
 * Deadline is a date in the format YYYY-MM-DD.
 * Deadline should not be before today's date.
 * Event name should be the **name of an already existing event**.
-* The command must contain all `td/`, `d/` and `ev/` for the command to be valid.
-* `TASK_DESCRIPTION`, `DEADLINE`, and `EVENT_NAME` must all be present.
 
 Examples:
-* `add_task td/Book Venue d/2023-12-23 ev/NUS Career Fair 2023` adds a `Book Venue` by `2023-12-23` task to the event `NUS Career Fair 2023`
+* `add_task td/Book Venue d/2023-12-23 ev/NUS Career Fair 2023` adds a `Book Venue` by `2023-12-23` task to the event `NUS Career Fair 2023`.
+
+### Viewing help : `help`
+
+Shows a message explaining how to access the help page.
 
+![help message](images/helpMessage.png)
+
+Format: `help`
```

Comment:
Can include a spacing between this line and the next line

Reply:
Done!